platform/darwin/src/MGLFeature.mm

```diff
@@ -42,6 +50,14 @@ - (id)attributeForKey:(NSString *)key {
     return self.attributes[key];
 }
 
+- (NSDictionary *)geoJSONFeatureDictionary {
```

Comment:
Rename `-geoJSONGeometryDictionary` to `-geoJSONDictionary` and `-geoJSONFeatureDictionary` to `geoJSONDictionary`. Then this method can call super’s implementation and wrap it in `NSDictionaryFeatureForGeometry()`.